source-academy/frontend
# Folders: (Draft) Add read-only workspace

### Description



This is part of an ongoing effort to prevent users from modifying a file that is not part of the question they are attempting. When a user accesses a file that is not part of the current question, the workspace is modified such that the content of the file cannot be changed and can only be read from.

### Type of change


### How to test


In `src\commons\assessmentWorkspace\AssessmentWorkspace.tsx`, change `isEditable` to `false` to enable read-only mode and `true` for default behaviour. Note that this sets all assessments to read-only mode for now.

### Checklist
